runtime/js/40_testing.js

```diff
@@ -3,10 +3,10 @@
 
 ((window) => {
   const core = window.Deno.core;
-  const { parsePermissions } = window.__bootstrap.worker;
   const { setExitHandler } = window.__bootstrap.os;
   const { Console, inspectArgs } = window.__bootstrap.console;
   const { metrics } = window.__bootstrap.metrics;
+  const { serializePermissions } = window.__bootstrap.permissions;
```

Comment:
Cool! Nice to see this being reused in that way, looks like we ended up with a single source of truth